llm_on_ray/inference/api_server_simple.py

```diff
@@ -22,11 +22,10 @@ def serve_run(deployments, model_list):
     for model_id, infer_conf in model_list.items():
         print("deploy model: ", model_id)
         deployment = deployments[model_id]
+
+        serve.start(http_options={"host": infer_conf.host, "port": infer_conf.port})
         serve.run(
             deployment,
-            _blocking=True,
```

Comment:
can we pass `keep_serve_terminal` and set it to the new `blocking` parameter in serve.run? Better to also rename `keep_serve_terminal` to `blocking` in our script which will be more consistent and easy to understand.

Reply:
Let's keep current logic for this PR. We printed something after this call. 
Use a separate PR to address the parameter:
In the new version, user should use Ctrl-C to stop the program after this blocking call. 
Maybe we can change the default behavior to blocking and use --serve_in_background